src/5.0_to_5.1/migrate_5.0_to_5.1.ts

```diff
@@ -35,22 +36,21 @@ export const migrate_50_to_51: MigrationFunction = (
     deserializeDashboardState,
     (
       dashboardState,
-      { dataModels, keysOfWidgetPluginsToRemove, onErrorWhileMigratingWidget },
```

Comment:
Do we still need `dataModels` in the type of the callbacks?

Reply:
Yes we do, otherwise you could not consume it with just the `MigrateXyzCallback` type.
But I'm not sure I understood your question